config.py

```diff
@@ -83,7 +83,8 @@ class Configuration(object):
     OVERDRIVE_INTEGRATION = "Overdrive"
     THREEM_INTEGRATION = "3M"
 
-    BASE_OPDS_AUTHENTICATION_DOCUMENT = "base_opds_authentication_document"
+    # ConfigurationSEtting key for a CDN's mirror domain
+    CDN_MIRROR_DOMAIN_KEY = u'mirror_domain'
```

Comment:
Looking at this in context, I think "mirrored_domain" is better, because "mirror_domain" is ambiguous whether it's the domain that you're mirroring or the domain _of_ the mirror.